Gopkg.toml

```diff
@@ -67,6 +67,10 @@
   name = "github.com/prometheus/client_golang"
   version = "1.1.0"
 
+[[constraint]]
+  name = "github.com/kubernetes-csi/csi-lib-utils"
+  version = "=v0.6.1"
```

Comment:
Is this a typo `=v`? It is a weird prefix that I have not seen before (and no such tag in the repo).

Reply:
this is required if you want to stick to the exact version